src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+/** 
+ * Store information for a contact
```

Comment:
This header comment can improve. Take note NOT to mention the child classes in the super class header comment.